find_pm_classes.py

```diff
@@ -0,0 +1,26 @@
+import ast
+import sys
+
+
+class Visitor(ast.NodeVisitor):
+    def __init__(self, file):
+        self.file = file
+        self.classes_used = set()
+
+    def visit_Call(self, node: ast.Call):
+        if isinstance(node.func, ast.Attribute):
+            if isinstance(node.func.value, ast.Name):
+                if node.func.value.id in {"pm", "pymc3"}:
+                    if node.func.attr[0].isupper():
+                        self.classes_used.add(node.func.attr)
```

Comment:
should also check if class was imported from PyMC3, e.g. `from pymc3 import Normal`